eos/form-factors/observables.cc

```diff
@@ -417,6 +418,39 @@ namespace eos
 
     // }}}
 
+    // B -> gamma
+    //}}}
```

Comment:
What is going on here?

Reply:
In the first place: A typo, I wanted to open a group :)

In the last review we wanted B->gamma* not in the B->V group.
So I think a group B->gamma containing the on-shell and off-shell group would make sense?
Otherwise, Should I leave it as only the B->gamma* group and we merge it with @pluegh's on-shell form factors later?

Reply:
Make it ``B -> gamma^(*)`` and we sort it later.